runner/run.go

```diff
@@ -230,10 +233,19 @@ func markAndFlush(runContext *TestRunContext, dest *ftwhttp.Destination, stageID
 		Version: "HTTP/1.1",
 	}
 
+	// check if destination host is an alias for localhost
+	addrs, err := net.LookupHost(dest.DestAddr)
+	if err != nil {
+		return nil, fmt.Errorf("ftw/run: can't resolve destination %+v: %w", dest, err)
+	}
+	// we only take the first IP address
+	if net.ParseIP(addrs[0]).IsLoopback() {
+		host = dest.DestAddr
```

Comment:
Apart from the comment, I think this should be set to the host entry of the current stage. The `Host` header will be the deciding factor for choosing the location / virtual host, so if we change the host here to `mylocalhostalias` but the test request has a host name of `anotheralias`, the two requests will end up in different locations / virtual hosts.

```suggestion
        // With multiple locations / virtual hosts, logs can be written to different files,
        // in which case the internal requests for marking and flushing must end up in the
        // same log file (host name match).
		host = dest.DestAddr
```

Reply:
Ping @fzipi

Reply:
Did you consider my proposal to use the host instead of the address that I had written above the suggestion?

Reply:
Yes. Or we make this configurable on the command line. In any case, I don't think that using the address field is the right choice. Or am I missing something?

Reply:
Ok, I think this version captures the idea better. Using the host header from the test is better. We default to "localhost" if test is raw/encoded.